### ISSUE TITLE
Anruftabelle in Notitzen unterhalb eigener Einträge oder eigenes Feld

### ISSUE THREAD

**Issue by xantippe** [Kruemelino/FritzBoxTelefon-dingsbums#15]
Könnte die Anruftabelle, die in Notizen angelegt wird, auch in ein eigenes Feld oder automatisch unterhalb der eigenen Eingaben starten?

**Comment by Kruemelino** [comment]
Ja, sicher. Bin aber gerade an einer anderen Baustelle dran. Kann mich da gerade nicht wirklich dami beschäftigen.
